wiki/Difficulties/osu!/Normal/en.md

```diff
@@ -1,9 +1,12 @@
+Normal
+=========
+
 Summary
 -------
 
-Normal [difficulties](difficulty) are difficulties in which you must have basic skills to play them. They are the easiest difficulties next to [Easy](Easy_(Difficulty)) difficulties.
+Normal [difficulties](/wiki/Difficulties) are difficulties in which you must have basic skills to play them. They are the easiest difficulties next to [Easy](/wiki/Difficulties/osu!/Easy) difficulties.
 
-[Mapping](Mapping) good Normal difficulties can appear to be hard at times, but after a while, it's pretty simple. Here's how to get things started.
+[Mapping](/wiki/Beatmapping) good Normal difficulties can appear to be hard at times, but after a while, it's pretty simple. Here's how to get things started.
 
 Guidelines
```

Comment:
I assume these aren't from some kind of Code of Conduct or Ranking **Guidelines**... I suggest to rename this heading to `Mapping Tips` or similar. (I will ignore any recursive occurrences)